app/views/articles/new.html.haml

```diff
@@ -3,6 +3,10 @@
     = form.label :title
     %br/
     = form.text_field :title, id: :article_title
+  %p
+    = form.label :author
+    %br/
```

Comment:
No need to add this field. Whoever’s logged in should be assigned as the author of the article. You can deal with it in the controller

Reply:
i thought that the requirements for the author part of the assignment was to be able to assign multiple authors to a piece. Would using an author form field be a bad way approaching this?